osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/MonitoringMapper.java

```diff
@@ -57,24 +54,24 @@ public void configure(final MapperFactory mapperFactory) {
          * in XML is the actual list of the objects to be mapped to the
          * "selectedValues" in the rest of the code and vice versa.
          */
-        mapperFactory
-                .classMap(ProfileGenericDataRequest.class,
-                        org.opensmartgridplatform.domain.core.valueobjects.smartmetering.ProfileGenericDataRequest.class)
-                .fieldAToB(SELECTED_VALUES_CAPTURE_OBJECT, SELECTED_VALUES)
-                .fieldBToA(SELECTED_VALUES, SELECTED_VALUES_CAPTURE_OBJECT).byDefault().register();
-
-        mapperFactory
-                .classMap(GetProfileGenericDataRequest.class,
-                        org.opensmartgridplatform.domain.core.valueobjects.smartmetering.ProfileGenericDataRequestData.class)
-                .fieldAToB(SELECTED_VALUES_CAPTURE_OBJECT, SELECTED_VALUES)
-                .fieldBToA(SELECTED_VALUES, SELECTED_VALUES_CAPTURE_OBJECT).byDefault().register();
+        mapperFactory.classMap(
+                org.opensmartgridplatform.adapter.ws.schema.smartmetering.monitoring.GetPowerQualityProfileRequestData.class,
+                org.opensmartgridplatform.domain.core.valueobjects.smartmetering.GetPowerQualityProfileRequest.class)
+                     .fieldAToB(SELECTED_VALUES_CAPTURE_OBJECT, SELECTED_VALUES)
+                     .fieldBToA(SELECTED_VALUES, SELECTED_VALUES_CAPTURE_OBJECT).byDefault().register();
+
+        mapperFactory.classMap(
+                org.opensmartgridplatform.adapter.ws.schema.smartmetering.bundle.GetPowerQualityProfileRequest.class,
+                org.opensmartgridplatform.domain.core.valueobjects.smartmetering.GetPowerQualityProfileRequest.class)
+                     .fieldAToB(SELECTED_VALUES_CAPTURE_OBJECT, SELECTED_VALUES)
+                     .fieldBToA(SELECTED_VALUES, SELECTED_VALUES_CAPTURE_OBJECT).byDefault().register();
 
         mapperFactory.getConverterFactory().registerConverter(new ObisCodeValuesConverter());
 
         mapperFactory.getConverterFactory().registerConverter(new ProfileEntryValueConverter());
 
-        mapperFactory.getConverterFactory().registerConverter(new ProfileGenericDataResponseConverter());
-        mapperFactory.getConverterFactory().registerConverter(new ProfileGenericDataConverter());
+        //mapperFactory.getConverterFactory().registerConverter(new PowerQualityProfileResponseDataConverter());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)